client/src/legacy/LeftAndMain.js

```diff
@@ -878,11 +889,15 @@ $.entwine('ss', function($) {
     /**
      * Save tab selections in order to reconstruct them later.
      * Requires HTML5 sessionStorage support.
+     *
+     * Parameters:
+     *  (String) url used for session storage key
+     *  (Boolean) resetTab true force selected tab to first, else current active
      */
-    saveTabState: function() {
+    saveTabState: function(url, resetTab) {
       if(typeof(window.sessionStorage)=="undefined" || window.sessionStorage === null) return;
 
-      var selectedTabs = [], url = window.ss.tabStateUrl();
+      var selectedTabs = [];
```

Comment:
```suggestion
      var selectedTabs = [];
      if (url === undefined) {
        const url = window.ss.tabStateUrl();
      }
```
Someone might be using this for their own purposes, so we should aim to retain backwards compatibility.